lib/views/auth/login.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:prephq_connect/views/mentor/mentorscreen.dart';
+import 'package:prephq_connect/views/student/studentscreen.dart';
+import 'package:mongo_dart/mongo_dart.dart' as md;
+
+
+import '../../registrationscreen.dart';
+
+class MyHomePage extends StatefulWidget {
```

Comment:
Can we name this `LoginScreen`? Makes it a little easier to immediately grasp what the widget is